README.md

```diff
@@ -1,9 +1,747 @@
+and setup instructions to guide you and assist you in any issues at you may encounter. This document
+will be revised periodically to contain updated to the environment and to the XPCoin software itself.**
+
+
+**Part 1: Building the windows wallet**
+
+This section will cover building the windows wallets. This will cover both the server daemon and the qt
+wallet version.
+
+**Pathing Requirements:**
+
+PATH is an environment variable on Unix-like operating systems, DOS, OS/2, and Microsoft Windows,
+specifying a set of directories where executable programs are located. In general, each executing
+process or user session has its own PATH setting.
+
+You will need to make sure these following items are added to the windows path.
+
+1. CMake is an open-source, cross-platform family of tools designed to build, test and package
+    software. CMake is used to control the software compilation process using simple platform and
+    compiler independent configuration files and generate native makefiles and workspaces that
+    can be used in the compiler environment of your choice. The suite of CMake tools were created
+    by Kitware in response to the need for a powerful, cross-platform build environment for open-
+    source projects such as ITK and VTK.
+
+
+```
+a. C:\Program Files\CMake\bin
+```
+2. devenv.exe is a process which belongs to the Microsoft Visual Studio, which is an application
+    development suite from Microsoft. This program is a non-essential system process but should
+    not be terminated unless suspected to be causing problems.
+       a. C:\Program Files (x86)\Microsoft Visual Studio\ 2017 \Community\Common7\IDE
+
+
+**Requirements:**
+
+Building the wallet will require you to have the following software packages. Please make sure that you
+have before compiling or running the batch files.
+
+1. Windows 10 Pro
+2. Visual Studio 2017 Community
+    a. https://www.visualstudio.com/thank-you-downloading-visual-
+       studio/?sku=Community&rel=
+3. Chocolatey
+    a. https://chocolatey.org/
+4. Qt 5.10.1: Open Source (Community Version)
+    a. https://www.qt.io/download
+    b. Install to HD location **C:\QT**
+5. Visual C++ Redistributable for Visual Studio 2015 (msvc2015 32bit)
+    a. https://www.microsoft.com/en-us/download/details.aspx?id=
+6. Windows SDK 8.1 and Windows 10 SDK (ver. 10.0.16299.91)
+    a. https://developer.microsoft.com/en-us/windows/downloads/sdk-archive
+7. GitHub CLI
+    a. https://git-scm.com/download/win
+
+**Getting the XPCoin Software:**
+
+The next steps will have to be performed in a command shell being run as administrator
+
+1. XPCoin GitHub: https://github.com/eXperiencePoints/XPCoin
+    a. git clone https://github.com/eXperiencePoints/XPCoin
+2. Add features required for build environment
+    a. cd XPCoin
+    b. git checkout feature/support_windows
+    c. git checkout feature/add_appveyor_script
```

Comment:
`feature/support_windows` and `feature/add_appveyor_script` already merged.
please use `git checkout develop`